swan/Dockerfile

```diff
@@ -95,11 +98,18 @@ ADD python/jupyter_server_config.py /home/${NB_USER}/.jupyter/jupyter_server_con
 
 USER root
 
+# Create directories where we will store:
+# - SWAN Jupyter extensions
```

Comment:
This is a bit misleading... There we also have the proxy server, simpervisor and, later, dask labextension (which are not "SWAN"), right?

Reply:
To say instead: directory where to store modules that are need by our extensions or our extensions themselves. This path gets removed from notebooks by swankernelenv.